src/puppetlabs/ezbake/core.clj

```diff
@@ -699,6 +707,38 @@ Additional uberjar dependencies:
 (defmulti action
   (fn [action & args] action))
 
+(defn munge-deps
+  ([deps]
+   (into {} (concat (for [[k v] deps]
+                      (let [[project-name version] k
+                            dependencies (munge-deps v)
+                            metadata (if (empty? dependencies) {"version" version} {"version" version "dependencies" dependencies })
+                            ]
+                        {(str project-name) metadata}))))))
+
+(defn generate-build-metadata-files
+  [lein-project]
+  (let [dep-map (deputils/generate-dependency-map lein-project)
+        deps (munge-deps dep-map)
+        metadata {"packaging_type" {"ezbake" (get-ezbake-version lein-project)}
+                  "git_sha" (get-git-sha)
+                  "build_time" (new java.util.Date)
```

Comment:
What will be reading this date value?

Will it be consumed by some other program?

The default format for dates is often times something human-readable, like `Tue Jun 27 17:54:19 PDT 2017`, but this is a pain for a program to parse.  Is the current format something like that?

If this is going to be consumed by some other program, it might better to leave the datetime as a big long like `87938104858` which is easily consumable in many languages.  It's only when we go to actually show the datetime in a UI to a human do we need to convert it to a human-readable format like `Tue Jun 27....`.